src/stores/FiltersHistoryStore.ts

```diff
@@ -113,43 +117,35 @@ class FiltersHistoryStore {
 	}
 
 	@action
-	public onEventFilterSubmit = async (newFilters: FiltersHistoryType<EventFilterState>) => {
-		if (isEmptyFilter(newFilters.filters)) return;
-
-		const { type, timestamp } = newFilters;
-		const equilizedFilter = getNonEmptyFilters(newFilters.filters);
-
-		if (this.eventsHistory.some(({ filters }) => isEqual(filters, equilizedFilter))) return;
-
-		const filter: FiltersHistoryType<EventFilterState> = {
-			timestamp,
-			type,
-			filters: equilizedFilter,
-			isPinned: false,
-		};
-
-		this.indexedDb.addDbStoreItem(IndexedDbStores.FILTERS_HISTORY, filter);
-		this.filterHistory = [...this.filterHistory, filter];
+	public onEventFilterSubmit = async (
+		newItem: FiltersHistoryType<EventFilterState>,
+		notify?: boolean,
```

Comment:
delete notify
this function returns promise so we can chain with `then` or await where this function is being called